index.bs

```diff
@@ -901,6 +914,12 @@ To <dfn>process an attributionsrc response</dfn> given a [=suitable origin=]
                 |destinationSite|, |reportingOrigin|, |privateStateToken|, and
                 |context|'s [=current wall time=].
             1. If |trigger| is not null, [=trigger attribution=] with |trigger|.
+        1. If |osSourceURL| is not null and the user agent supports OS
```

Comment:
Do we expect there to be steps here other than "pass this to the OS"? Looking at Web Bluetooth, they use phrasing like:

"If the UA is running on a system that has a Bluetooth radio", so we could potentially be more direct, but for now I think this is fine especially since there is no spec we could point to like in the case of bluetooth.